Only run orig file PHPCS linting when the new version has lint issues

david-binda opened the pull request:
When there are no linting issues reported by PHPCS in the new file, there is no point in checking the orig verion, since we would not report any issues anyway.

This should speed up the phpcs-changed for files w/o lint issues.

props to @sirbrillig for the idea.

sirbrillig reviewed (commented):
This looks great! Nice job; I thought it would be harder to refactor for this behavior.

The only thing I'd like to do is have some tests to prove this behavior (mainly to guard against future regressions).

david-binda commented:
Yep, good call! I have added a new test for git workflow, and adjusted existing one for svn workflow. I made sure the test are properly failing w/o the logic change from this PR, and are passing when the change is applied.

sirbrillig commented:
Thank you!
